### PR TITLE
v0.21.2

### PR DESCRIPTION
**Pull request by icidasset**
Couple of bugfixes, see CHANGELOG and commits for info.

### CODE REVIEW

File: src/data-root.ts [hunk 1]
```diff
@@ -85,9 +85,6 @@ export async function update(
   if (fetchController) fetchController.abort()
   fetchController = new AbortController()
 
-  // Ensure peer connection
-  await ipfs.reconnect()
```
Comment (walkah) [inline src/data-root.ts:89]: this is no longer needed?

File: README.md [hunk 2]
```diff
@@ -56,6 +56,18 @@ const state = await wn.initialise({
       publicPaths: [ "Mixtapes" ]
     }
   }
+
+}).catch(err => {
```
Comment (walkah) [inline README.md:60]: 👍

File: src/fs/filesystem.ts [hunk 3]
```diff
@@ -210,6 +210,9 @@ export class FileSystem {
     if (!sameTree) {
       throw new Error("`mv` is only supported on the same tree for now")
     }
+    if (await this.exists(to)) {
```
Comment (walkah) [inline src/fs/filesystem.ts:213]: good addition!

**Review by walkah — APPROVED** [review]
Looks good! 🎉 🕺 💃

File: src/data-root.ts (hunk 1 above)
Comment (icidasset) [reply src/data-root.ts:89]: Is handled in https://github.com/fission-suite/auth-lobby/blob/master/src/Javascript/Worker.js#L61 now.

**Pull request merged**